packages/mobile/src/qrcode/QRScanner.tsx

```diff
@@ -23,50 +24,31 @@ interface DispatchProps {
   handleBarcodeDetected: typeof handleBarcodeDetected
 }
 
-interface State {
-  isScanningEnabled: boolean
-}
-
 type Props = DispatchProps & WithTranslation & NavigationFocusInjectedProps
 
-class QRScanner extends React.Component<Props, State> {
+class QRScanner extends React.Component<Props> {
   static navigationOptions = () => ({
     ...headerWithBackButton,
     headerTitle: i18n.t('sendFlow7:scanCode'),
   })
 
   timeout: undefined | number = undefined
 
-  state = {
-    isScanningEnabled: true,
-  }
+  state = {}
 
   camera: RNCamera | null = null
 
   // This method would be called multiple times with the same
   // QR code, so we need to catch only the first one
   onBardCodeDetected = (rawData: any) => {
```

Comment:
Nit: not a new typo but "onBarcodeDetected" (removing d)

Reply:
Fixed!